int16 Raw
float32 Volt
---
int16 State